abb_irb6640_support/CHANGELOG.rst

```diff
@@ -0,0 +1,51 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This package, although mostly a copy (content wise) of another, has not yet been released, so it seems strange to include the changelog of another pkg here.